resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -65,6 +65,7 @@ private[spark] object Constants {
   val ENV_JAVA_OPT_PREFIX = "SPARK_JAVA_OPT_"
   val ENV_CLASSPATH = "SPARK_CLASSPATH"
   val ENV_DRIVER_BIND_ADDRESS = "SPARK_DRIVER_BIND_ADDRESS"
+  val ENV_DRIVER_POD_NAME = "SPARK_DRIVER_POD_NAME"
```

Comment:
We had better put ths line should be before `ENV_DRIVER_BIND_ADDRESS`. Could you spin off this `ENV_DRIVER_BIND_ADDRESS` addition to a new PR, @pan3793 ?

Reply:
Sure, opened https://github.com/apache/spark/pull/39160 for this.